src/main/java/org/mule/extension/validation/internal/CommonValidationOperations.java

```diff
@@ -251,6 +257,27 @@ public void isTime(String time, @Optional String locale, @Optional String patter
     validateWith(new TimeValidator(time, nullSafeLocale(locale), pattern, context), context);
   }
 
+  /**
+   * Validates that {@code since} did not occur more than {@code time} amount of {@code timeUnit} units
```

Comment:
why do you hate the user? Don't think of extension's jdocs like technical information. The extension's documentation are generated from them. Write them for the user. This is a lot more clear if instead you say `Validates the amount of time that has elapsed since the moment in the {@code since} parameter is greater than an specified amount of {@code time}`